### DISCUSSION TITLE
Welcome to the Nettle Magic Project Discussions!

### DISCUSSION THREAD

**Discussion by nettlep** [nettlep/magic#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions, get clarification.
  * Share ideas - are you marking decks? Share your experience!
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by nelsonnil** [comment]
Hi, how do you marks the deck?

**Reply by nettlep** [reply]
Section 6 of the documentation is dedicated to marking the deck:

https://nettlep.github.io/magic/

**Reply by nelsonnil** [reply]
My purpose is to buy the stamp and use non-invisible black ink and use a press to keep the deck from moving.  I do not understand why you say it is very laborious.  How long does it take to make a deck?

**Reply by nettlep** [reply]
I don't see how the press would be used, unless you plan to create a stamp that is the exact size of the deck, and you plan to stamp the entire deck at once. I've was unable to get anything like that to work well. If you manage to get it to work, please post your results and technique!

The stamp I used was laborious because each card is stamped individually. This is especially true for invisible inks, which dry very quickly on the stamp, requiring just the right amount of ink on the stamp so that it doesn't bleed, but also doesn't dry before you can get the card onto the stamp. This usually requires re-wetting the stamp every card or two before you can continue stamping.

Black ink is far less labor intensive (you wet the stamp once and use that for the entire deck.) But it still takes me 10 minutes to mark a deck on both ends.

When I mark a deck, I also test it for robust readability. There's a feature on the app to show error correction, which highlights cards that are poorly marked (and hence, are difficult to read.) See this section of the docs for details:

https://nettlep.github.io/magic/#marking/examiningthequalityofadeck

**Reply by nelsonnil** [reply]
Could you send email so i can give you some suggestions and queries. Elmagonil@gmail.com , Iam professional magician and developer

**Comment by nelsonnil** [comment]
something similar to this: https://www.penguinmagic.com/p/S12202

**Comment by alexisheloir** [comment]
Hi there,
Looks like some folks in China can provide customized decks [1]. Do you think it would worth the try ? Has anyone tried already ?
[1] : https://fr.aliexpress.com/item/1005009051338268.html
